packages/thunderstore-api/src/apiFetch.ts

```diff
@@ -81,14 +97,14 @@ export async function apiFetch(props: {
   }
 
   const { config, path, request, queryParams, useSession = false } = args;
+  const configSnapshot = config();
```

Comment:
What's the idea behind adding this variable? Seems the config is used only on the next line so there doesn't seem to a possibility of this function using different values if the config is changed outside? And if the point is to protect from changes during the whole fetch process, shouldn't the snapshot be taken earlier than this point? Again, introducing an extra variable is just a nitpick thing, but it seems this might be done based on incorrect assumptions.

Reply:
Good catch, that variable was actually a remnant from a thing I tried 👍